consul/aio.py

```diff
@@ -22,7 +22,8 @@ def __init__(self, *args, loop=None, connections_limit=None,
                                          **connector_kwargs)
         session_kwargs = {}
         if connections_timeout:
-            session_kwargs['timeout'] = connections_timeout
+            timeout = aiohttp.ClientTimeout(total=connections_timeout)
```

Comment:
This is weird, since aiohttp is supposedly polymorph there.
See https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client.py#L383-L386